warnings_config.yml

```diff
@@ -0,0 +1,20 @@
+sphinx:
+  enabled: true
+  min: 0
+  max: 0
+  exclude:
+    - 'WARNING: Connection failed: HTTPSConnectionPool\(host=.+, port=\d+\): Max retries exceeded with url: /api/v2/[\w/?=&\\]+ \(Caused by NameResolutionError\(\"<urllib3\.connection\.HTTPSConnection object at [0-9a-z]+>: Failed to resolve .+ \(\[Errno -2\] Name or service not known\)\"\)\)'
```

Comment:
Do we maybe need here also Errno -3 from above?

Reply:
Added in 3364edd005c0468d72c99f3c043bd95c1648ed6b